### PR TITLE
fix send button not disabled bug when switch between MFAs

### PR DESCRIPTION
**Pull request by yumingcao-okta**
issue:
bug

fix:
disable

[OKTA-89746] (https://oktainc.atlassian.net/browse/OKTA-89746)

@rchild-okta

### CODE REVIEW

File: test/spec/MfaVerify_spec.js [hunk 1]
```diff
@@ -1764,6 +1765,20 @@ function (Q, _, $, Duo, OktaAuth, LoginUtil, CryptoUtil, Util, MfaVerifyForm, Be
...
+      itp('Verify Push after switching from Google TOTP', function () {
+        return setupGoogleTOTP({'features.autoPush': true})
+          .then(function (test) {
+            spyOn(CookieUtil, 'isAutoPushEnabled').and.returnValue(true);
+            test.beacon.dropDownButton().click();
+            test.beacon.getOptionsLinks().eq(0).click();
+            return tick(test);
+          })
+          .then(function (test) {
+              var button = test.form.submitButton();
```
Comment (rchild-okta) [inline test/spec/MfaVerify_spec.js]: nit: spacing

**Comment by rchild-okta** [comment]
btw, looks like there's a lint failure from the travis run

**Comment by rchild-okta** [comment]
Overall looks good - couple nits but after fixing 🚢

**Comment by yumingcao-okta** [comment]
@rchild-okta nit fixed

**Comment by rchild-okta** [comment]
🚋

**Pull request merged**